rhosp10/profiles/stig.profile

```diff
@@ -0,0 +1,53 @@
+documentation_complete: true
+
+title: 'RHOSP STIG'
```

Comment:
@redhat-rmcallis please change this to `[DRAFT] STIG for Red Hat OpenStack Plaform 10`